finat/point_set.py

```diff
@@ -68,6 +68,43 @@ def expression(self):
         return gem.Literal(self.point)
 
 
+class UnknownPointSingleton(AbstractPointSet):
+    """A point set representing a single point with unknown location but known
+    `gem.Variable` expression.
```

Comment:
This is not the right way to link things, if you want fixed width, you need two ` at each end.